### DISCUSSION TITLE
[Bug] 打开页面会提示3个错误，如下

### DISCUSSION THREAD

**Discussion by huicewang** [lobehub/lobehub#873] [Development Question] [answered]
### 💻 系统环境 | Operating System

Windows

### 🌐 浏览器 | Browser

Chrome

### 🐛 问题描述 | Bug Description

![image](https://github.com/lobehub/lobe-chat/assets/19237481/f5b7ae52-136f-4709-ba06-f9140fe77540)
![image](https://github.com/lobehub/lobe-chat/assets/19237481/e3985be6-9663-4071-9c4a-d4fe68d8bdf7)
![image](https://github.com/lobehub/lobe-chat/assets/19237481/fbd38fc9-3590-41a9-8054-863323a44993)

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
应该只是第一次吧？后续都不会有了

[a comment by an automated account is not shown]

**Comment by huicewang** [comment]
是的，关闭之后就没有了

[a comment by an automated account is not shown]
